nss-c31-unusual-prestige/nutshell
# Av save input

# Description

Messages now print to dom with their username attached. New messages post when the "send" button is clicked. 

Applies to issue # 5

## Type of change

Please delete options that are not relevant.


# Testing Instructions

-git fetch --all
--add the data from the "json.messages.example" api if you don't have any messeges in the file you use for testing.
-run your grunt server.
-see that messages appear along with the username of the user who posted the message.
-type a new message into the input box.
-click "send"
-confirm that the new message posts to the dom.
-NOTE: If you don't have at least 4 users defined in the API, some will return "undefined". It's not a bug, just a result of features that haven't been implemented yet.
# Checklist:
